### PR TITLE
feat(curriculum): test clearing of previous arrays in Sorting Visualizer

### PR DESCRIPTION
**Pull request by gikf**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

- - -
- Adds test for clearing previously generated numbers and/or sorted arrays.
- This gives better (specific) indication why different test might be failing, while _everything_ appear to be working fine.
- Without such feature implemented in camper solution, currently fails only one test, which in this case does not give any hints regarding the issue:
  > After you click `#sort-btn`, each `div` within `#array-container` should contain five `span`, each with a number as its text, and arranged to represent the steps required by Bubble Sort algorithm to sort the starting array.

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/lab-sorting-visualizer/6716249b5405164036fd0b0d.md [hunk 1]
```diff
@@ -207,6 +208,25 @@ Array.from(children).forEach(el => {
 })
 ```
 
+When `#starting-array` already contains generated array, or `#array-container` contains sorted array, clicking the `#generate-btn` should remove other elements in the `#array-container`, leaving only `#starting-array` with newly generated numbers.
```
Comment (Dario-DC) [inline curriculum/challenges/english/25-front-end-development/lab-sorting-visualizer/6716249b5405164036fd0b0d.md:211]:
```suggestion
When `#starting-array` already contains a generated array, or `#array-container` contains the sorted array, clicking the `#generate-btn` should remove other elements in the `#array-container`, leaving only `#starting-array` with newly generated numbers.
```

File: curriculum/challenges/english/25-front-end-development/lab-sorting-visualizer/6716249b5405164036fd0b0d.md [hunk 2]
```diff
@@ -27,6 +27,7 @@ Fulfill the user stories below and get all the tests to pass to complete the lab
 1. You should have a function named `highlightCurrentEls` that takes an HTML element and a numeric index.
 1. The `highlightCurrentEls` function should set the `border` of the given element's child at the given index, and the child immediately after the index, to have a `dashed` style, a `red` color, and a width of your choice.
 1. When you click `#generate-btn` you should use the `fillArrContainer` function to fill `#starting-array` with five `span` elements, each with a random number as its text.
+1. When you click `#generate-btn` the `#array-container` should contain `#starting-array` with generated numbers. Other elements should be removed from `#array-container`.
```
Comment (Dario-DC) [inline curriculum/challenges/english/25-front-end-development/lab-sorting-visualizer/6716249b5405164036fd0b0d.md:30]:
```suggestion
1. When you click `#generate-btn` you should use the `fillArrContainer` function to fill `#starting-array` with five `span` elements, each with a random number as its text. If present, other elements should be removed from `#array-container`.
```

**Review by Dario-DC — COMMENTED** [review]
Thank you for catching that. I would add a few words to the current user story instead of adding a new one. But other than that it looks good to me :+1:

**Review by Dario-DC — APPROVED** [review]
LGTM :+1:

**Review by moT01 — APPROVED** [review]

**Pull request merged**